enterprise/backend/src/metabase_enterprise/workspaces/driver/common.clj

```diff
@@ -0,0 +1,34 @@
+(ns metabase-enterprise.workspaces.driver.common
+  "Common utilities for workspace isolation drivers."
+  (:require
+   [clojure.string :as str]
+   [metabase.system.core :as system]))
+
+;;;; Naming
+
+;; re-using https://github.com/metabase/metabase/pull/61887/commits/c92e4a9cc451c61a13fef19ed9d6107873b17f07
+;; (original ws isolation code)
+(defn- instance-uuid-slug
+  "Create a slug from the site UUID, taking the first character of each section."
+  [site-uuid-string]
+  (apply str (map first (str/split site-uuid-string #"-"))))
+
+(defn isolation-schema-name
```

Comment:
General note on "schema":

Sameer was careful to use the new term "namespace" to refer to the generalization of the schema concept, so it can apply to clickhouse's database or a datalake's partition concepts just as well.

I wonder whether we should also adopt this more generic term? Maybe the term schema has already set in too deeply (e.g. that it's what we use for CH database in `metabase_table`)

Just thought I'd get your thoughts 😄